eng/packaging.targets

```diff
@@ -202,6 +202,18 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="WarnOnProjectReferenceToFrameworkAssemblies"
+          BeforeTargets="IncludeTransitiveProjectReferences"
+          Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)' AND
```

Comment:
nit: we use lower case 'and' and 'or' statements in this file (for the most part) and in most places in src/libraries.